force-app/main/default/lwc/inlineMessage/inlineMessage.js

```diff
@@ -9,7 +10,7 @@ const VARIANTS = {
 
 export default class InlineMessage extends LightningElement {
     /** Generic / user-friendly message */
-    @api message;
+    @api friendlyMessage = 'Error retrieving data';
```

Comment:
You need to change consumers of this component as well. In `productFilter.html` for example,

`<c-inline-message friendly-message="Error loading categories" errors={categories.error}></c-inline-message>`.

This was caught by the tests running in CI btw :). It's a little hard to catch because of the other log noise but if you scroll up in the CI error you'll see an `UnhandledPromiseRejectionWarning`.

cc/ @kevinv11n